rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -385,4 +427,31 @@ impl FirmwareProcessor {
 
         (nb, nf)
     }
+
+    /// Extend measurement into PCR1
+    /// # Arguments
+    /// * `pcr_bank` - PCR Bank
+    /// * `sha384` - SHA384
+    /// * `txn` - Mailbox Receive Transaction
+    /// # Returns
+    /// * `()` - Ok
+    ///     Err - StashMeasurementReadFailure
+    fn extend_measurement(
+        pcr_bank: &PcrBank,
+        sha384: &mut Sha384,
+        txn: &mut MailboxRecvTxn,
+    ) -> CaliptraResult<()> {
+        const MEASUREMENT_WORDS: usize =
+            core::mem::size_of::<StashMeasurement>() / core::mem::size_of::<u32>();
+        let mut slice: [u32; MEASUREMENT_WORDS] = [0u32; MEASUREMENT_WORDS];
+        txn.copy_request(&mut slice)?;
+
+        let measurement = StashMeasurement::read_from(slice.as_bytes())
+            .ok_or(CaliptraError::FW_PROC_STASH_MEASUREMENT_READ_FAILURE)?;
+
+        // Extend measurement into PCR1.
+        pcr_bank.extend_pcr(PcrId::PcrId1, sha384, measurement.measurement.as_bytes())?;
+
```

Comment:
will the event log come in a separate PR?

Reply:
Will be part of this PR.

Reply:
Added.